libraries/websphere_base.rb

```diff
@@ -347,26 +346,17 @@ def cluster_exists?(clus_name)
       end
 
       # returns true if server name is a member of specified cluster for a specified node.
-      def member?(clus_name, serv_name, serv_node)
+      def member?(clus_name, serv_name, serv_node, bin_directory = bin_dir)
```

Comment:
I'd suggest that we do this instead:
```
      def member?(clus_name, serv_name, serv_node, bin_directory = '/opt/IBM/WebSphere/AppServer/bin')
        cookbook_file "#{bin_directory}/cluster_member_exists.py" do
          cookbook 'websphere'
          source 'cluster_member_exists.py'
          mode '0755'
          action :create
        end

        # Get all server members from the cluster
        Chef::Log.info("Checking for: #{clus_name} #{serv_node} #{serv_name}")
        cmd = "./cluster_member_exists.py #{clus_name} #{serv_node} #{serv_name}"
        mycmd = wsadmin_returns(cmd, bin_directory)
```

Reply:
@chrisminton done; don't know why it works now but didn't originally..... I give up!

Reply:
@chrisminton bump!